firebase-functions/functions/src/models/room.ts

```diff
@@ -1,5 +1,7 @@
-﻿import { GameType } from './game-type';
+﻿import { Game } from './game';
+import { GameType } from './game-type';
 
 export interface Room {
   typeOfGame?: GameType;
+  games: Game[];
```

Comment:
I think it is much better to have this hierarchical structure now instead of the seperate entities :)